analytics.go

```diff
@@ -232,11 +236,24 @@ func (c *Client) send(msgs []interface{}) {
 		return
 	}
 
+	for i := 0; i < 10; i++ {
```

Comment:
10 seems like a reasonable default? Could potentially retry it indefinitely as well.